Tasks/ServiceFabricDeploy/deploy.ps1

```diff
@@ -31,6 +31,7 @@ try {
     $compressPackage = [System.Boolean]::Parse((Get-VstsInput -Name compressPackage))
     $skipUpgrade =  [System.Boolean]::Parse((Get-VstsInput -Name skipUpgradeSameTypeAndVersion))
     $skipValidation =  [System.Boolean]::Parse((Get-VstsInput -Name skipPackageValidation))
+    $unregisterUnusedVersions = [System.Boolean]::Parse((Get-VstsInput -Name unregisterUnusedVersions))
```

Comment:
Not able to run test.

![image](https://cloud.githubusercontent.com/assets/11148158/24489305/2e4ed69c-153a-11e7-8555-25411476f64f.png)